modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -938,14 +938,15 @@ export class Wallet implements IWallet {
         );
       }
 
-      const value = this.spendableBalanceString();
-      if (_.isUndefined(value) || value === '0') {
+      const value = await this.maximumSpendable();
```

Comment:
Add tests for the change

Reply:
It looks like we cant write test case for sweep functionality, @gianchandania any idea if theres a way?

Reply:
> It looks like we cant write test case for sweep functionality

Can you elaborate on why?

Reply:
Added tests 😄